classes/rental.rb

```diff
@@ -0,0 +1,20 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] As requested by the previous reviewer to add book and person to `attr_acessor`  to allow access to viewing and adding books and person 👍🏽so kindly update it to be `attr_accessor` instead of `attr_reader` 🔥🙏🏻✅

Reply:
@rotimiazeez thanks for the review. However, I want to point out that `attr_accessor` will not work here because I already manually added a setter method for the `book` and `person` properties.

Reply:
@preciousbetine I see.